src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,9 @@
+.catalog {
+  // gap: 48px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
```

Comment:
Here too. You can add another class main__catalog and include margins there

Reply:
really, I can implement mixes, thanks!